packages/core/package.json

```diff
@@ -9,10 +9,13 @@
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "scripts": {
-    "build": "run-s build:cjs build:esm build:css",
+    "build": "run-s build:cjs build:esm build:css build:tokens",
     "build:cjs": "tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
     "build:css": "node ./build",
-    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm"
+    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
+    "build:tokens": "ts-node ./src/tokens/index.ts",
```

Comment:
As far as I can tell, src/tokens/index.ts doesn't have any sideEffects. What is actually built here/there?

Reply:
Artifact I need to remove from when I was trying something else